//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// abcefg_T_AP.h
//
// Code generation for function 'abcefg_T_AP'
//

#ifndef ABCEFG_T_AP_H
#define ABCEFG_T_AP_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void abcefg_T_AP(double P_init, double V_init, double A_init, double P_wayp,
                 double A_wayp, double A_max, double A_min, double J_max,
                 double J_min, double T, creal_T t[14]);

#endif
// End of code generation (abcefg_T_AP.h)
